odxtools/utils.py

```diff
@@ -60,3 +65,33 @@ def is_short_name_path(test_val: str) -> bool:
     See also: ISO 22901 section 7.3.13.3
     """
     return _short_name_path_pattern.fullmatch(test_val) is not None
+
+
+@overload
+def resolve_snref(target_short_name: str,
```

Comment:
I think this method should be in `odxlink` file

Reply:
the thought was that `odxlink.py` contains the infrastructure required for the `ODXLINK` mechanism as specified by the standard (i.e., `FOO-REF ID="bar"` tags) while short name references are a entirely different mechanism (one with awful semantics, cf. #200). If you believe it is better to move the function there, I'll do so...

Reply:
odxlink seems more suitable/related than a generic utils file

Reply:
okay, done: https://github.com/mercedes-benz/odxtools/pull/299/commits/ac56647d3aaa7bb014269ad7ef1c285055e35495